tests/driver/api.test.ts

```diff
@@ -841,6 +841,7 @@ describe('Mongoose Model API level tests', async () => {
             res = await Vector.
                 findOne({}).
                 sort({ $vector: { $meta: [99, 1] } });
+            //@ts-ignore
```

Comment:
Is this necessary? Does this mean that `Vector.findOne().sort()` isn't returning an object that has a `name` property?

Reply:
Yes. 
Since, `res` is defined as type `any` above, without `//@ts-ignore` here, I'm seeing below error
```
TS2339: Property 'name' does not exist on type 'any[]'.
```

Reply:
Strange, I'll take a look, but `// @ts-ignore` is fine for now

Reply:
See #141